worker/uniter/operation/runhook.go

```diff
@@ -195,10 +198,14 @@ func (rh *runHook) afterHook(state State) (_ bool, err error) {
 		if !isLeader || err != nil {
 			return hasRunStatusSet && err == nil, err
 		}
-		rel, err := ctx.Relation(rh.info.RelationId)
-		if err == nil && rel.Suspended() {
+		rel, rErr := ctx.Relation(rh.info.RelationId)
+		if rErr == nil && rel.Suspended() {
 			err = rel.SetStatus(relation.Suspended)
 		}
+	case hooks.PreSeriesUpgrade:
+		logger.Debugf("completing pre upgrade series hook. updating state of series upgrade.")
+		err = rh.callbacks.SetUpgradeSeriesStatus(params.UnitCompleted)
+		// Does the unit status need to be set to something here?
```

Comment:
valid comment?

Reply:
Oh gee, just noticed I borked that. Will fix

Reply:
That is, will fix once, I find out what the status should be set to. Right now I don't know